.travis.yml

```diff
@@ -27,24 +39,28 @@ before_install:
 install:
   # Upgrade pip and setuptools. Mock has issues with the default version of
   # setuptools
-  - |
-    pip install --upgrade pip
-    pip install --upgrade setuptools
-    pip install $NUMPY
-    pip install $MPL
-    pip install -r requirements.txt
+  - pip install --upgrade pip
+  - pip install --upgrade setuptools
+
+  - pip install $NUMPY
+  - pip install $MPL
```

Comment:
I added freetype and libpng with APT in another branch.  The build was less than 10% faster, which probably has more to do with system load than any build time gains.